cmd/syncthing/main.go

```diff
@@ -592,7 +602,7 @@ func syncthingMain(runtimeOptions RuntimeOptions) {
 	// We want any logging it does to go through our log system.
 	mainService := suture.New("main", suture.Spec{
 		Log: func(line string) {
-			l.Debugln(line)
+			l.Infoln(line)
```

Comment:
I think it's pointless to have this in debug, disagree?

Reply:
Currently agree, because unsolved #4758.